### ISSUE TITLE
[EN][NPC] Ethereum Jailor

### ISSUE THREAD

**Issue by Achiver** [Atlantiss/BugTracker#4469]
<a href="http://www.wowhead.com/npc=23008/ethereum-jailor"> Ethereum Jailors </a> dont spawn at all. I've been looking for them all day and I haven't seen any.
Bildo said it's been fixed in <a href="https://github.com/Atlantiss/BugTracker/issues/4162"> #4162 </a> but NO , they still dont spawn.

**Comment by pelekon** [comment]
Yup, Bildo just forgot about this fix, I'll send him priv message about it.

**Comment by TranceAccount** [comment]
@Bildo

**Comment by aendry** [comment]
still broken
